### PR TITLE
Adds cloak hoods, moves cloaks back to the exosuit slot.

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
:cl: TechnoAlchemist
add: Cloaks now have hoods, toggled using an action button.
tweak: Cloaks are now worn in the exosuit slot.
tweak:  Cloaks have armor values, and can hold items reflecting their department.
/:cl:

When cloaks were first added back in the day I intended for them to be cool alternatives to armor, not capes. There was some confusions because cloaks usually have hoods or something that make them seem like an actual garment and not just an accessory for an outfit. 

So, I've moved the cloaks to the exosuit slot, and given them hoods to reflect their intended purpose.

### CODE REVIEW

File: code/modules/clothing/suits/cloaks.dm [hunk 1]
```diff
@@ -18,43 +18,103 @@
 	flags_1 = NODROP_1
 	flags_inv = HIDEHAIR|HIDEEARS
 
-/obj/item/clothing/neck/cloak/suicide_act(mob/user)
+/obj/item/clothing/suit/hooded/cloak/suicide_act(mob/user)
 	user.visible_message("<span class='suicide'>[user] is strangling [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return(OXYLOSS)
 
-/obj/item/clothing/neck/cloak/hos
+/obj/item/clothing/suit/hooded/cloak/hos
```
Comment (kevinz000) [inline code/modules/clothing/suits/cloaks.dm]: Please make sure I can carry the HoS egun!!
Please?

Comment (TechnoAlchemisto) [reply code/modules/clothing/suits/cloaks.dm]: o fuck lol i forgot to let him hold guns

Comment (optimumtact) [reply code/modules/clothing/suits/cloaks.dm]: no

Comment (TechnoAlchemisto) [reply code/modules/clothing/suits/cloaks.dm]: ^ he be holding guns

**Comment by WJohn** [comment]
Can we see the hoods in game? Having them as their own pieces separated from the cloaks is hard to gauge.

**Comment by Supermichael777** [comment]
Winter is coming

File: code/modules/clothing/suits/cloaks.dm [hunk 2]
```diff
@@ -18,43 +18,101 @@
...
 	name = "head of security's cloak"
 	desc = "Worn by Securistan, ruling the station with an iron fist."
 	icon_state = "hoscloak"
+	hoodtype = /obj/item/clothing/head/hooded/cloakhood/hos
+	allowed = list(/obj/item/melee/baton,/obj/item/melee/classic_baton,/obj/item/grenade,/obj/item/gun,/obj/item/reagent_containers/spray/pepper,/obj/item/restraints/handcuffs,/obj/item/device/assembly/flash/handheld,/obj/item/clothing/glasses,/obj/item/ammo_casing/shotgun,/obj/item/ammo_box,/obj/item/reagent_containers/food/snacks/donut,/obj/item/reagent_containers/food/snacks/donut/jelly,/obj/item/kitchen/knife/combat,/obj/item/device/flashlight/seclite,/obj/item/melee/classic_baton/telescopic,/obj/item/device/radio,/obj/item/clothing/gloves/,/obj/item/restraints/legcuffs/bola)
 
-/obj/item/clothing/neck/cloak/qm
+/obj/item/clothing/head/hooded/cloakhood/hos
+	icon_state = "hoshood"
+	armor = list(melee = 30, bullet = 30, laser = 30, energy = 10, bomb = 25, bio = 0, rad = 0, fire = 70, acid = 90)
```
Comment (optimumtact) [inline code/modules/clothing/suits/cloaks.dm]: this is really absurd

File: code/modules/clothing/suits/cloaks.dm [hunk 3]
```diff
@@ -18,43 +18,101 @@
...
 	desc = "Worn by the commander of Space Station 13."
 	icon_state = "capcloak"
+	armor = list(melee = 30, bullet = 35, laser = 35, energy = 10, bomb = 25, bio = 0, rad = 0, fire = 50, acid = 90)
+	allowed = list(/obj/item/disk, /obj/item/stamp, /obj/item/reagent_containers/food/drinks/flask, /obj/item/gun, /obj/item/melee, /obj/item/storage/lockbox/medal, /obj/item/device/assembly/flash/handheld, /obj/item/storage/box/matches, /obj/item/lighter, /obj/item/clothing/mask/cigarette, /obj/item/storage/fancy/cigarettes, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman)
+	hoodtype = /obj/item/clothing/head/hooded/cloakhood/cap
 
-/obj/item/clothing/neck/cloak/hop
+/obj/item/clothing/head/hooded/cloakhood/cap
+	icon_state = "caphood"
+	armor = list(melee = 30, bullet = 35, laser = 35, energy = 10, bomb = 25, bio = 0, rad = 0, fire = 50, acid = 90)
```
Comment (optimumtact) [inline code/modules/clothing/suits/cloaks.dm]: taking the piss

File: code/modules/clothing/suits/cloaks.dm (hunk 2 above)
Comment (TechnoAlchemisto) [reply code/modules/clothing/suits/cloaks.dm]: yeah that fire and acid resistance is too high

File: code/modules/clothing/suits/cloaks.dm (hunk 3 above)
Comment (TechnoAlchemisto) [reply code/modules/clothing/suits/cloaks.dm]: funny yellow drink

**Comment by JohnOxford** [comment]
I feel like they'll be under used now, because they were always supplements to whatever armor the heads wore. The RD's tele armor, the HOS's trench coat, the Captain Carapace. ect

**Comment by Potato-Masher** [comment]
Yeah, good point. Why would I want to wear these in the Exosuit slot over the armored trench coat or the Cap's carapace (other than looking sexy)?

**Comment by Potato-Masher** [comment]
I wonder, do you think it's possible to have the cloaks fit in both slots?

File: code/modules/clothing/suits/cloaks.dm (hunk 1 above)
Comment (kevinz000) [reply code/modules/clothing/suits/cloaks.dm]: <3

**Comment by Mey-Ha-Zah** [comment]
Maybe give the capes some armor value... or maybe they can attach ONTO a pre-existing exosuit, like medals on jumpsuit.

Because, so far, I only ever see 1 person wearing a cloak if it replaces armor slot, and thats QM.

**Comment by Mey-Ha-Zah** [comment]
Also, you have Hop's cloak hood hold his gear instead of the cloak... whoops?

**Comment by TechnoAlchemisto** [comment]
lol u wild

**Comment by Spaceinaba** [comment]
yall are fucking pussies it's a fair trade off 
stats are traded off for
_**A E S T H E T I C S**_

**Comment by Spaceinaba** [comment]
I can guarantee you I will still be wearing the RD's cloak 24/7 because that shit is SNAZZY and if you'd rather trade that for some pussy gay shit ass armor that makes you teleport you can shove it up your ugly ASS

**Comment by Durkel** [comment]
The idea of a bunch of man children running on board a  space station with their arms held out while their capes flail behind them is as sad as it is amusing.

**Comment by JohnMcLaime** [comment]
Give players an option to choose? Make so we can switch modes for them i.e. fashion - goes to neck slot with no armor and storage and armor which is what you suggest.

**Comment by Tlaltecuhtli** [comment]
cape is worse than hos armor

**Review by WJohn — CHANGES_REQUESTED** [review]
Please improve the lighting on the hoods, especially on the side sprites. I'm really not sold on how they look.

Or optionally, redesign the cloaks entirely because they all look fucking hideous anyway.

**Comment by ExcessiveUseOfCobblestone** [comment]
> people wearing armor for armor values and not for looks

lol

**Comment by Aloraydrel** [comment]
Armor values meaning anything when stuns exist 🙌

**Comment by TechnoAlchemisto** [comment]
sorry i got 2 downvotes time to kms

**Comment by JohnOxford** [comment]
no

**Pull request closed without merging**